<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourcePanel</class>
 <widget class="QMainWindow" name="ResourcePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="webGISCreationMessageWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: yellow</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;a href=&quot;https://my.nextgis.com/signup&quot;&gt;Create Web GIS at nextgis.com&lt;/a&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="webGISCreationMessageCloseLable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;a href=&quot;close&quot; style=&quot;text-decoration:none&quot;&gt;x&lt;/a&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="nrw_reorces_tree_container"/>
    </item>
    <item>
     <widget class="QLabel" name="helpMessageLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;&quot;/&gt;&lt;a href=&quot;help&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Help&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddAsGeoJSON"/>
   <addaction name="actionAddWFS"/>
   <addaction name="separator"/>
   <addaction name="actionOpenMapInBrowser"/>
   <addaction name="separator"/>
   <addaction name="actionCreateNewGroup"/>
   <addaction name="actionImportQGISProject"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionAddAsGeoJSON">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add GeoJSON</string>
   </property>
   <property name="toolTip">
    <string>Add vector layer as GeoJSON</string>
   </property>
  </action>
  <action name="actionAddWFS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add WFS</string>
   </property>
   <property name="toolTip">
    <string>Add WFS layers</string>
   </property>
  </action>
  <action name="actionCreateNewGroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create new group</string>
   </property>
   <property name="toolTip">
    <string>Create new resource group</string>
   </property>
  </action>
  <action name="actionImportQGISProject">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import current project</string>
   </property>
   <property name="toolTip">
    <string>Import layers from current project</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionOpenMapInBrowser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open map in browser</string>
   </property>
   <property name="toolTip">
    <string>Open map in browser</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
